core/Twig.php

```diff
@@ -74,6 +76,11 @@ function piwik_escape_filter(Twig_Environment $env, $string, $strategy = 'html',
         case 'url':
             $encoded = rawurlencode('{');
             return str_replace('{{', $encoded . $encoded, $string);
+        case 'whole_url':
+            if (!UrlHelper::isLookLikeSafeUrl($string)) { // if this could be a dangerous link, replace it w/ an empty url
+                return 'javascript:;';
```

Comment:
would this need to be `javascript:void(0)`?

Reply:
I guess it could be? I'm not sure if it would change anything (assuming it's for `preventDefault()`?).